mobile/ios/Podfile.lock

```diff
@@ -299,13 +299,13 @@ PODS:
   - React-jsinspector (0.65.1)
   - react-native-encrypted-storage (4.0.2):
     - React-Core
-  - react-native-image-picker (4.9.0):
+  - react-native-image-picker (4.10.0):
```

Comment:
This and other dependencies have been updated here, but not in the `package.json` this causes next time somebody runs `pod install` it overrides these again to the previous version

Reply:
hmmm good to know. Why did it update these then I wonder?

Reply:
I just won't check it in for now.